README.md

```diff
@@ -39,11 +39,11 @@ npm start
 
 # In another shell, run grafton.
 grafton test --product=bonnets --plan=small --region=aws::us-east-1 \
-    --client-id=$CLIENT_ID \
-    --client-secret=$CLIENT_SECRET \
-    --connector-port=$CONNECTOR_PORT \
+    --client-id=21jtaatqj8y5t0kctb2ejr6jev5w8 \
+    --client-secret=3yTKSiJ6f5V5Bq-kWF0hmdrEUep3m3HKPTcPX7CdBZw \
+    --connector-port=3001 \
```

Comment:
This is fine to set since `CONNECTOR_PORT` isn't an ENV var used in the application